### PR TITLE
Added possibility to choose app for open link, converted autostart to…

### PR DESCRIPTION
**Pull request by bogomazdmitry**
… background (if it is marked in teams app)

### CODE REVIEW

File: src/MMT.UI/App.xaml.cs [hunk 1]
```diff
@@ -13,5 +11,75 @@ namespace MMT.UI
...
+
+            if (isOwned)
+            {
+                FirstAppOnStart(e);
+            }
+            else
+            {
+                NotFirstAppOnStart(e);
+            }
+
```
Comment (BlazeOfSoul) [inline src/MMT.UI/App.xaml.cs]: Remove 'tab' pls

File: src/MMT.Core/StaticResources.cs [hunk 2]
```diff
@@ -43,6 +43,14 @@ public static string UpdateExe
                 return @"AppData\Local\Microsoft\Teams\Update.exe";
             }
         }
+        
+        public static string LaunchExe
+        {
+            get
+            {
+                return @"AppData\Local\Microsoft\Teams\current\Teams.exe";
```
Comment (BlazeOfSoul) [inline src/MMT.Core/StaticResources.cs:51]: Is this a standard way? Can't it be shortened?

**Comment by bogomazdmitry** [comment]
In progress pull request

File: src/MMT.Core/StaticResources.cs (hunk 2 above)
Comment (bogomazdmitry) [reply src/MMT.Core/StaticResources.cs:51]: @BlazeOfSoul It is ok xD

**Comment by bogomazdmitry** [comment]
Pull request is ready to merge IMO

File: src/MMT.UI/MainWindow.xaml.cs [hunk 3]
```diff
@@ -201,17 +201,24 @@ private void Hyperlink_RequestNavigate(object sender, RequestNavigateEventArgs e
 
         private async void LstProfiles_MouseDoubleClick(object sender, MouseButtonEventArgs e)
```
Comment (BlazeOfSoul) [inline src/MMT.UI/MainWindow.xaml.cs:202]: CamelCase and snake_case in one?

**Pull request closed without merging**